index.html

```diff
@@ -0,0 +1,339 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <section class="menu-mobile">
+      <div id="menu-container" class="mobile-menu-container">
+        <div class="bg-img"></div>
+        <button class="close-btn">x</button>
+        <ul class="mobile-menu-ul" id="anchor-tags">
+          <li><a href="#works">Portfolio</a></li>
+          <li><a href="#about-myself">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
+    </section>
+    <header class="main-header">
+      <nav>
+        <ul class="header-menu">
+          <li><a href="#" class="info-text">Melkamu</a></li>
+          <li>
+            <a href="#" id="hamburger-btn" class="nav__button"
+              ><i
+                ><img
+                  src="Asset/Icon - Menu.png"
+                  alt="hamburger-icon-menu"
+                  class="info-image"
+                  id="hamburger-btnicon" /></i
+            ></a>
+          </li>
+          <li class="desktop-menu-container">
+            <ul class="menu-lists">
+              <li class="item"><a href="#portfolio">Portfolio</a></li>
+              <li class="item"><a href="#about">About</a></li>
+              <li class="item"><a href="#contact">Contact</a></li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
```

Comment:
- [ ] Kindly note that for your desktop adaptation the media query should be similar to this


```
@media screen and (min-width: 1080px) { 
//your styling code
}
```

Kindly note that a precise measurement of 1080 pixels is not mandatory, as any value ranging from 768 to 1440 pixels would be acceptable.